pycbc/workflow/datafind.py

```diff
@@ -906,6 +906,8 @@ def run_datafind_instance(cp, outputDir, connection, observatory, frameType,
     # workflow format output file
     cache_file = File(ifo, 'DATAFIND', seg, extension='lcf',
                       directory=outputDir, tags=currTags)
+    cache_file.PFN(cache_file.cache_entry.path, site='local')
+    
```

Comment:
Yes. Does declaring a PFN for the cache files in all cases have any negative affects? I would think not.